src/open_samus_returns_rando/files/schema.json

```diff
@@ -665,7 +665,6 @@
                 "ITEM_RESERVE_TANK_LIFE",
                 "ITEM_RESERVE_TANK_MISSILE",
                 "ITEM_RESERVE_TANK_SPECIAL_ENERGY",
-                "ITEM_OFFWORLD",
```

Comment:
I'm in a rush: Why is this removed? We still have a generic offworld model which is the RDV logo icon thing, don't we?

Reply:
Because I remembered that offworld models use `ITEM_NONE`, so this item_id is technically never used. Obviously we could use it in the tests, but None with 0 quantity works just fine.

Reply:
Generic offworld is unaffected

Reply:
Oh, OK. So, it is `ITEM_NONE` just with a different model. Totally makes sense

Reply:
Yep